bazel/cc_wrapper.py

```diff
@@ -71,6 +71,10 @@ def main():
   else:
     argv += sys.argv[1:]
 
+  # Enforce LLD if LLD is in argv, by default the latest -fuse-ld will be used
+  if "-fuse-ld=lld" in argv:
```

Comment:
in -> **not** in?

Reply:
No this is intended to be `in`. Bazel will add `-fuse-ld=gold` in some cases, gcc/clang will take the last -fuse-ld argument, so whenever we specified lld once, add it to last.

Reply:
OMG. Thank you for the explanation!

Reply:
Please do update the comments with this :)